application/src/main/java/org/thingsboard/server/service/ws/DefaultWebSocketService.java

```diff
@@ -843,6 +851,17 @@ private void sendPing() {
                 }));
     }
 
+    private void cleanupClosedSessions() {
+        wsSessionsMap.keySet().forEach(sessionId -> {
+            if (!msgEndpoint.isOpen(sessionId)) {
+                log.debug("[{}] Session does not exists, removing all subscriptions", sessionId);
```

Comment:
This shouldn't normally happen, so let's change the log level to info. 
We also would like to know tenant id and user id from the log.
Also, consider changing the message to smth like "Cleaning up stale session".